requirements/base.in

```diff
@@ -5,6 +5,6 @@
 django-celery
 future
 celery>= 3.1.25,<4.0	
-Django >= 1.11
+Django >= 1.11, < 2.0
```

Comment:
assuming that we want to continue using `Django 1.11` and use the latest version only for testing

Reply:
The problem is that this causes a version conflict when trying to test with Django 2.0+ in edx-platform.  We should handle this like we do in other repos: have `make upgrade` move the testing Django dependency to a separate file, and let tox control the version.

Reply:
the last `make upgrade` we did, was executed on `python2` and it installed the latest supported version of Django,  now if we run `make upgrade`(with `python3`), it installs `django2.2.x` as the base dependency, my understanding is the base version would still remain `1.11.x`, we can control the version by specifying it in constraints as done in some other repos or it's specified in `edx-platform`. 
As per my understanding, `tox` is testing all Django versions from `1.11.x` to `2.2.x`, could you specify any repo (as you mentioned above "other repos") to follow?

Reply:
setup.py [derives](https://github.com/edx/edx-celeryutils/blob/master/setup.py#L81) `install_requires` from this file, so adding the constraint here tells pip in repositories using this library that Django 2.0+ cannot be successfully installed together with it.  edx-platform is one example of the setup I have in mind:

* [make upgrade](https://github.com/edx/edx-platform/blob/master/Makefile#L93-L96)
* [tox.ini](https://github.com/edx/edx-platform/blob/master/tox.ini#L71-L73)